diff --git a/advance/advance.mak b/advance/advance.mak
index a42c31d..10ea867 100644
--- a/advance/advance.mak
+++ b/advance/advance.mak
@@ -27,16 +27,16 @@ INSTALL_BINFILES += $(OBJ)/$(EMUNAME)$(EXE)
 INSTALL_MANFILES += $(DOCOBJ)/advmame.1
 INSTALL_MANFILES += $(DOCOBJ)/advdev.1
 ifeq ($(CONF_EMU),mame)
-INSTALL_DATAFILES += $(srcdir)/support/event.dat
-INSTALL_DATAFILES += $(srcdir)/support/history.dat
-INSTALL_DATAFILES += $(srcdir)/support/hiscore.dat
-INSTALL_ROMFILES += $(srcdir)/contrib/mame/free/rom/gridlee.zip
-INSTALL_ROMFILES += $(srcdir)/contrib/mame/free/rom/polyplay.zip
-INSTALL_ROMFILES += $(srcdir)/contrib/mame/free/rom/robby.zip
-INSTALL_SAMPLEFILES += $(srcdir)/contrib/mame/free/rom/gridlee.zip
-INSTALL_SNAPFILES += $(srcdir)/contrib/mame/free/snap/gridlee.zip
-INSTALL_SNAPFILES += $(srcdir)/contrib/mame/free/snap/polyplay.zip
-INSTALL_SNAPFILES += $(srcdir)/contrib/mame/free/snap/robby.zip
+#INSTALL_DATAFILES += $(srcdir)/support/event.dat
+#INSTALL_DATAFILES += $(srcdir)/support/history.dat
+#INSTALL_DATAFILES += $(srcdir)/support/hiscore.dat
+#INSTALL_ROMFILES += $(srcdir)/contrib/mame/free/rom/gridlee.zip
+#INSTALL_ROMFILES += $(srcdir)/contrib/mame/free/rom/polyplay.zip
+#INSTALL_ROMFILES += $(srcdir)/contrib/mame/free/rom/robby.zip
+#INSTALL_SAMPLEFILES += $(srcdir)/contrib/mame/free/rom/gridlee.zip
+#INSTALL_SNAPFILES += $(srcdir)/contrib/mame/free/snap/gridlee.zip
+#INSTALL_SNAPFILES += $(srcdir)/contrib/mame/free/snap/polyplay.zip
+#INSTALL_SNAPFILES += $(srcdir)/contrib/mame/free/snap/robby.zip
 endif
 ifeq ($(CONF_EMU),mess)
 INSTALL_DATAFILES += $(srcdir)/support/sysinfo.dat
